src/UserManager.php

```diff
@@ -162,11 +166,15 @@ static function (array $user) use ($userIdClaim) {
         $users
       );
 
-    $this->logger->debug(json_encode($users, JSON_PRETTY_PRINT));
-
     $users = $this->userStorage->loadByProperties([$userIdField => $userIdsToKeep]);
     foreach ($users as $user) {
-      $this->logger->info($this->t('Retaining user @name.', ['@name' => $user->label()]));
+      $this->logger->info(
+        'Retaining user @name.',
+             [
+               '%name' => $user->label(),
```

Comment:
The placeholder (`@…`) does not match the replacement key (`%…`). I think we should go for `@` (cf. https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Component%21Render%21FormattableMarkup.php/function/FormattableMarkup%3A%3AplaceholderFormat/11.x) all over the place.